### PR TITLE
Fix seed ts compile options - Add tab section for mac and windows examples

### PR DESCRIPTION
**Pull request by elsangedy**

### CODE REVIEW

File: content/300-guides/050-database/300-seed-database.mdx [hunk 1]
```diff
@@ -156,7 +156,7 @@ Here we suggest some specific seed scripts for different situations. You are fre
 
    ```ts
    "prisma": {
-     "seed": "ts-node --compiler-options \"{\\\"module\\\":\\\"commonjs\\\"}\" prisma/seed.ts"
+     "seed": "ts-node --compiler-options {\"module\":\"commonjs\"} prisma/seed.ts"
```
Comment (janpio) [inline content/300-guides/050-database/300-seed-database.mdx:159]: I am not confident this would work on Windows out of the box.

Comment (elsangedy) [reply content/300-guides/050-database/300-seed-database.mdx:159]: The current documentation doesn't work for macos.
Maybe we can put these two options? (for windows, for unix)

Comment (janpio) [reply content/300-guides/050-database/300-seed-database.mdx:159]: The current documentation was chosen as it was supposed to work on all platforms. If that is not the case any more, we need to find an option that does.

Is there an issue for this not working?

**Comment by Jolg42** [comment]
Closing because duplicate, done with https://github.com/prisma/docs/pull/2338 https://prisma-company.slack.com/archives/CGMCB82N8/p1631898183074400

**Pull request closed without merging**